### ISSUE TITLE
String vs number and publication year

### ISSUE THREAD

**Issue by vojtechhuser** [OHDSI/MedlineXmlToDatabase#7]
I would like to get nice numerical date for each PMID. My colleague told me that the XML data can be messy and conversion to number may not be without errors.

1.
are all fields imported to database as string by design? (I think so) 

2.
Did other users have experience with using  medcit.datecreated_year as the publication year for a given article?

**Comment by schuemie** [comment]
Yes, dates are messy in the MEDLINE XML. This is why the publication date is automatically parsed (as best as possible) to fill the table `pmid_to_date`.

The parsing logic can be found [here].(https://github.com/OHDSI/MedlineXmlToDatabase/blob/master/src/org/ohdsi/medlineXmlToDatabase/PmidToDate.java)
